webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/datasource/AdditionalAxis.scala

```diff
@@ -6,7 +6,7 @@ import com.scalableminds.util.tools.{Box, Failure, Full}
 import play.api.libs.json.{Format, Json}
 
 // bounds: lower bound inclusive, upper bound exclusive
```

Comment:
The Array hashcode stuff is annoying, good catch! 
Maybe a test that calculates the hashcode of a datasource would be nice?